packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-alternate-flows/main/index.md

```diff
@@ -27,14 +27,14 @@ In this scenario, the org is set up in the following manner:
 
 1. The org is initially configured following the steps described in:
```

Comment:
We can remove the stack selector tags from this page as there is nothing now that is language-specific.

Reply:
OK. I'll have to remove the stack selector structure and convert this back to a one-page guide.